### DISCUSSION TITLE
Questions about the scaling ratio of web pages.

### DISCUSSION THREAD

**Discussion by wyek1n** [transitive-bullshit/nextjs-notion-starter-kit#262] [General]
Hello, thank you for developing such a great tool!
I used it to deployed a web [navigation](http://wye.cool), but it shows that the default size is so large that it is appropriate to use a zoom of about 70% in PC (Chrome) or 50% in mobile（Safari）.
Where can I change the default display scale for device access?
Thank you again! ～
![Snipaste_2022-04-09_09-02-40](https://user-images.githubusercontent.com/96416488/162550475-0efd1878-4ac1-4648-83b2-4c406273d2ea.jpg)
![Snipaste_2022-04-09_09-03-42](https://user-images.githubusercontent.com/96416488/162550478-50280525-c520-4464-be1a-4b798d19fdf2.jpg)

**Comment by transitive-bullshit** [comment]
Hey @wyek1n 👋 

By default this project uses the same font sizes as notion and many other blog sites like substack and medium.com.

Of course, if you want to customize it that's the whole point of using an open source solution 😄 

I would recommend changing the default font size of notion in `styles/notion.css`. As an example:

```css
.notion {
  font-size: 14px;
}
```

For other things like the max-width of the page, you'll have to explore and tweak the CSS yourself.

Cheers
